crt_portal/cts_forms/templates/forms/confirmation.html

```diff
@@ -189,7 +189,10 @@ <h2 class='h1__display'>{% trans "Your submission" %}</h2>
 {% endblock footer_extra %}
 
 {% block page_js %}
-  <script src="https://touchpoints.app.cloud.gov/touchpoints/a24aeb5e/js" async></script>
   <script src="{% static 'js/modal.js' %}"></script>
   <script src="{% static 'js/redirect-modal.js' %}"></script>
 {% endblock %}
+
+{% block touchpoint_js %}
```

Comment:
Looks like the implementation in `report_primary_complaint.html` here differs by one character so it doesn't extend this block. 
```suggestion
{% block touchpoints_js %}
```